pkg/u8g2/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+USEMODULE += xtimer
+
+FEATURES_REQUIRED += periph_gpio
+
+CFLAGS += -DU8X8_WITH_USER_PTR
```

Comment:
This should go in a separate `Makefile.include` file.

Reply:
Moved.